src/js/actions/search/commands.js

```diff
@@ -60,12 +61,10 @@ define(function (require, exports) {
                 }
 
                 if (menuTree.$MENU) {
-                    resultPath += menuTree.$MENU + ">";
+                    resultPath += menuTree.$MENU + " > ";
```

Comment:
I realize this isn't your doing, but: do you know where _else_ this separator is used? It seems like something that should be defined once. Actually, even better if this separator is localized so that it could be different in different languages. (Who knows if `>` makes sense in German or Japanese?)